crates/astria-sequencer/Cargo.toml

```diff
@@ -41,8 +47,10 @@ divan = { workspace = true, optional = true }
 futures = { workspace = true }
 hex = { workspace = true, features = ["serde"] }
 ibc-types = { workspace = true, features = ["with_serde"] }
+indexmap = { workspace = true }
 penumbra-ibc = { workspace = true, features = ["component", "rpc"] }
 metrics = { workspace = true }
+pbjson-types = { workspace = true }
```

Comment:
You likely need this for timestamps?  Since this is integral to types inside `astria-core`, we can reexport it there so you don't need to declare the dependency here.